test/utils/image/manifest.go

```diff
@@ -68,6 +69,7 @@ func initReg() RegistryList {
 	registry := RegistryList{
 		GcAuthenticatedRegistry: "gcr.io/authenticated-image-pulling",
 		DockerLibraryRegistry:   "docker.io/library",
+		DockerIORegistry:        "docker.io",
```

Comment:
with new dockerhub limitations we are trying to avoid it if possible

Reply:
Good point ... is there another registry we can keep an iperf3 in?

Reply:
`"gcr.io/kubernetes-e2e-test-images"`

Reply:
Pinging @claudiubelu for help with this one. Claudiu configured the image promotion for test images used for conformance in https://github.com/kubernetes/kubernetes/tree/master/test/images .
Think we should be able to piggy back off of the automation built for that.

Reply:
In the past, we used to have an ``iperf`` image, but we've centralized it into agnhost since then. Is there a specific reason you require iperf3? If iperf2 is sufficient, you can simply use the agnhost image instead. If not, you can add it to the agnhost image in ``kubernetes/test/images/agnhost/Dockerfile``, and also the ``Dockerfile_windows`` file too, IMO.

Reply:
iperf2 doesn't seem to be in the agnhost image:

```
✗ docker run --rm -it --entrypoint sh k8s.gcr.io/e2e-test-images/agnhost:2.21
/ # iperf2
sh: iperf2: not found
```

so I'll try adding iperf3 instead.

Reply:
nvm -- misunderstood iperf, `iperf -v` shows that iperf2 is indeed in agnhost 👍 , and covers our use case so agnhost as-is should be fine!